docs/sources/observability-as-code/provision-resources/intro-git-sync.md

```diff
@@ -63,12 +63,14 @@ With Git Sync, you can make changes to the files in the provisioned folder in Gi
 
 ## Known limitations
 
-Git Sync is under development and the following limitations apply:
+**Git Sync is under development and the following limitations apply:**
 
 - You can only sync dashboards and folders. Refer to [Supported resources](#supported-resources) for more information.
   - If you're using Git Sync in Grafana OSS and Grafana Enterprise, some resources might be in an incompatible data format and can't be synced.
+- Full-instance sync is not available in Grafana Cloud and has limitations in Grafana OSS and Grafana Enterprise. Refer to [Choose what to synchronize](../git-sync-setup.md#choose-what-to-synchronize) for more details.
 - You can only authenticate in GitHub using your Personal Access Token token.
 - Support for native Git, Git app, and other providers, such as GitLab or Bitbucket, is on the roadmap.
+- You cannot import, copy, move, or save resources using Git Sync. Instead, open a PR to carry out any of these tasks.
```

Comment:
Copy, move or save is about the existing resources that are not managed by Git Sync. The only want to make them to be managed by Git Sync currently it's to export it as a JSON, and commit them to the repository.

This makes me think we need a section to describe how to deal with existing resources. See this document as reference: https://docs.google.com/document/d/1qDP781AwrMjWRYsr1e-n91meaSEpOEVAoEN9ujRZcpc/edit?tab=t.0#heading=h.5l07qu3tkvmu